packages/expo/src/Notifications/Notifications.ts

```diff
@@ -438,4 +441,52 @@ export default {
     }
     return ExponentNotifications.setBadgeNumberAsync(number);
   },
+
+  async scheduleNotificationWithCalendarAsync(
+    notification: LocalNotification,
+    options: {
+      year?: number;
+      month?: number;
+      hour?: number;
+      day?: number;
+      minute?: number;
+      second?: number;
+      weekDay?: number;
+      repeat?: boolean;
+    } = {}
+  ): Promise<string> {
+    const areOptionsValid: boolean =
+      (options.month == null || isInRange(options.month,1,12)) &&
+      (options.day == null || isInRange(options.day, 1, 31)) &&
+      (options.hour == null || isInRange(options.hour, 0, 23)) &&
+      (options.minute == null || isInRange(options.minute, 0, 59)) &&
```

Comment:
Is comparing to `null` convenient for Expo developers? I think default value for empty entries in maps is `undefined`, so to call this method, if I'm right, I would have to:
```
scheduleNotificationWithCalendarAsync(…, {
  hour: 9, // the only property I want to set
  second: null,
  hour: null,
  // null for every option
})
```
instead of just `{ hour: 9 }`?

Reply:
https://playcode.io/320582?tabs=console&script.js&output

Reply:
Ah, didn't see that those are double equals signs. My eyes are too used to seeing `===`